src/utils.js

```diff
@@ -0,0 +1,110 @@
+export const Shuffle = (array) =>{
+    for (var k = 0; k < array.length; k++) {
+        var i = array[k].length;
+        if (i === 0)
+            return false;
+        else {
+            while (--i) {
+                var j = Math.floor(Math.random() * (i + 1));
+                var tempi = array[k][i];
+                var tempj = array[k][j];
+                array[k][i] = tempj;
+                array[k][j] = tempi;
+            }
+        }
+    }
+    return array
+}
+
+export const rando = () =>{
+        const colors = ['purple', 'orange', 'red', 'green', 'yellow', 'blue', 'white', 'blue']
```

Comment:
So all of these color names inside a string are "magic constants", and you should work to minimize the number of places that they show up in the code base. Currently (I think) they're here and in the board's default state -- and the more places that they are, the more places you have to update if we need to change the list for some reason.

Reply:
Also, why does blue show up twice?